app/models/Extraction.scala

```diff
@@ -121,16 +124,21 @@ object ExtractorInfo {
     def reads(json: JsValue): JsResult[URL] = JsSuccess(new URL(json.toString()))
     def writes(url: URL): JsValue = Json.toJson(url.toExternalForm)
   }
+
   implicit val dateFormat = new Format[Date] {
+    val simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
```

Comment:
doesn't look like we use simplDateFormat

Reply:
Thanks. I switched to using the function in `util.Parsers` at the last minute. Will fix.

Reply:
Removed.